app/ui/browser/actions/main-actions.js

```diff
@@ -34,7 +34,16 @@ export function createTab(location: ?string = undefined,
     const id = uuid.v4();
 
     // Start loading a tab while asking the User Agent to start its browsing session.
-    dispatch({ type: types.CREATE_TAB, id, ancestorId, location, options, instrument: true });
+    // Dispatching a list ensures only one render and prevents racing to set focus across user
```

Comment:
We may want to rethink the underlying approach (existing before this PR) to clearing the UI state. My thinking is that if we need to ensure only one render happens, we've lost idempotence.